src/pages/ScoreReport.vue

```diff
@@ -894,6 +901,12 @@ const exportAll = async () => {
         ? _get(assessment, 'scores.computed.composite')
         : _get(assessment, `scores.computed.composite.${rawScoreKey}`);
       tableRow[`${taskDisplayNames[taskId]?.name ?? taskId} - Support Level`] = support_level;
+      if (assessment.reliable !== undefined && !assessment.reliable && assessment.engagementFlags !== undefined) {
+        tableRow[`${taskDisplayNames[taskId]?.name ?? taskId} - Flags`] = _lowerCase(
```

Comment:
Would it be possible to get the reliability/engagement columns to be grouped with the scores? Right now the columns start with displaying all the score data first and then group all the engagement flags together at the end. I feel that it would be more clear if the engagement flags/reliability is presented with the scores. 

While we're at it, would it be possible to add a column the indicates reliability? (e.g.`${taskId} - Reliable`). The current implementation does not have much indication on whether or not a score is reliable, and I feel that that is important in context of adding the flags. Also, what do you think of changing the column name from `${taskId} - Flags` to `${taskId} - Engagement Flags` to better match the language in the table?

Reply:
Good point, let me add reliable next to the reliable ones, and change the column header